app/frontend/src/components/Settings/Settings.tsx

```diff
@@ -198,7 +198,8 @@ export const Settings = ({
                 onChange={(_ev?: React.FormEvent<HTMLElement | HTMLInputElement>, option?: IDropdownOption) => onChange("includeCategory", option?.key || "")}
                 aria-labelledby={includeCategoryId}
                 options={[
-                    { key: "", text: t("labels.includeCategoryOptions.all") }
+                    { key: "", text: t("labels.includeCategoryOptions.all") },
+                    { key: "__NONE__", text: t("labels.includeCategoryOptions.none") }
```

Comment:
I'm not 100% sure that this will be intuitive to folks as the way to go into "non-RAG" mode. Before looking at the PR, I was expecting a top-level checkbox like "Use sources for answers". But it's also nice to avoid adding even more things to settings. Do any other folks have thoughts?

Reply:
Given my use case, this approach is preferable. We will move this setting out of the developer settings and make it available to all users. This way, they will have a single dropdown to select a relevant subset of the documents in the index.

Reply:
No, in this repository, it will remain a developer setting. However, in our clone, we plan to migrate the dropdown to the UI, making it accessible to all users.

By the way, I’m curious to know which organization you're working with, as we also need to comply with regulations like DPIAs as a governmental entity.

Reply:
Good point by @EMjetrot, this feature is significant enough that there should be an option to enable/disable, as many organizations won't want to enable a general purpose chat. Given that, and concerns about unintuitive nature of the "None" dropdown, I'd vote for either a separate developer setting or a setting in the user space. (We don't have an obvious place for it, probably next to User uploads, but it's getting crowded. Perhaps you have ideas)

Reply:
My preference would be to remove the text next to the buttons, move the 'Upload File' button to the left of the chat input container, change the 'Remove Chat' button to 'New Chat,' and place it in the top-left (similar to the ChatGPT interface). The RAG switch could either be placed to the left of the question input container or in the top-right.

Reply:
* Re "upload file": I'd be slightly concerned that folks would think the file was *only* for that particular question, since that's often a feature of chat interfaces, versus persistent. 
* "Remove chat" -> "New chat" seems like a good change, given that "Remove chat" doesn't actually remove from chat history for folks using that. That's also the phrasing used by GitHub Copilot Chat.
* Placing in top-left to match ChatGPT could work. I don't feel strongly either way.
* I do like when there's text next to icons, as I can be a bit "icon-blind" at times, but I realize that's not always feasible. It's fine as long as we keep accessible tooltips.

cc @zedhaque as I think he's thought through this for the mobile-optimized design